### PR TITLE
[MRG]Portable debug

### LINKED ISSUE #355: RuntimeError: CUDA error (3): initialization error (google colab)
### Bug Description
When i exported model with method ImageClassifier, i get an exception 

100%|██████████| 243/243 [00:04<00:00, 54.86it/s]
100%|██████████| 275/275 [00:05<00:00, 46.61it/s]
(518, 64, 64, 3) (518, 1)
Preprocessing the images.
Preprocessing finished.

Initializing search.
Initialization finished.

+----------------------------------------------+
|               Training model 0               |
+----------------------------------------------+
Process ForkProcess-8:
Traceback (most recent call last):
  File "/usr/lib/python3.6/multiprocessing/process.py", line 258, in _bootstrap
    self.run()
  File "/usr/lib/python3.6/multiprocessing/process.py", line 93, in run
    self._target(*self._args, **self._kwargs)
  File "/usr/local/lib/python3.6/dist-packages/autokeras/search.py", line 277, in train
    raise e
  File "/usr/local/lib/python3.6/dist-packages/autokeras/search.py", line 270, in train
    verbose=verbose).train_model(**trainer_args)
  File "/usr/local/lib/python3.6/dist-packages/autokeras/nn/model_trainer.py", line 77, in __init__
    self.model.to(self.device)
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/modules/module.py", line 379, in to
    return self._apply(convert)
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/modules/module.py", line 185, in _apply
    module._apply(fn)
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/modules/module.py", line 191, in _apply
    param.data = fn(param.data)
  File "/usr/local/lib/python3.6/dist-packages/torch/nn/modules/module.py", line 377, in convert
    return t.to(device, dtype if t.is_floating_point() else None, non_blocking)
RuntimeError: CUDA error (3): initialization error

### PR DESCRIPTION
**Pull request by boyuangong**
This pull request resolves #355.

I checked this locally, and found no error.

Also added a tutorial for how to export keras model and how to load the exported model.

### CODE REVIEW

File: mkdocs/docs/start.md [hunk 1]
```diff
@@ -81,10 +81,18 @@ which can be directly feed into the `fit` function of `ImageClassifier`.
 
     from autokeras import ImageClassifier
     clf = ImageClassifier(verbose=True, augment=False)
-    clf.load_searcher().load_best_model().produce_keras_model().save('my_model.h5')
+    clf.export_autokeras_model(model_file_name)
```
Comment (jhfjhfj1) [inline mkdocs/docs/start.md]: This is not correct. It is export keras model.
You should keep the original and write a new section "how to export Auto-Keras model"

File: autokeras/utils.py [hunk 2]
```diff
@@ -171,10 +170,8 @@ def validate_xy(x_train, y_train):
 
 def read_csv_file(csv_file_path):
     """Read the csv file and returns two separate list containing file names and their labels.
-
```
Comment (jhfjhfj1) [inline autokeras/utils.py]: according to the style guide, these line should be kept.

File: mkdocs/docs/start.md (hunk 1 above)
Comment (boyuangong) [reply mkdocs/docs/start.md]: This has been fixed. Add another section as:
* How to export Portable model
* How to load exported Portable model?

File: autokeras/utils.py (hunk 2 above)
Comment (boyuangong) [reply autokeras/utils.py]: Fixed the docstring style

**Comment by boyuangong** [comment]
This is ready.

File: mkdocs/docs/start.md [hunk 3]
```diff
@@ -122,6 +133,7 @@ This is not specific to AutoKeras, however, the following will generate a .PNG v
     from keras.utils import plot_model
     plot_model(model, to_file='my_model.png')
 
+        
 # CnnGenerator tutorial
```
Comment (jhfjhfj1) [inline mkdocs/docs/start.md]: This should be CnnModule

**Pull request merged**